src/common/components/Recaptcha/RecaptchaWrapper.tsx

```diff
@@ -1,18 +1,23 @@
 import angular from 'angular'
-import { GoogleReCaptchaProvider } from 'react-google-recaptcha-v3'
 import { react2angular } from 'react2angular'
-import React from 'react'
+import React, { useMemo } from 'react'
 import { Recaptcha } from './Recaptcha'
 
 const componentName = 'recaptchaWrapper'
 
+declare global {
+  interface Window {
+      grecaptcha: any
+  }
+}
+
 interface RecaptchaWrapperProps {
   action: string
   onSuccess: (componentInstance: any) => void
   onFailure: (componentInstance: any) => void
   componentInstance: any
   buttonId: string
-  buttonType?: 'submit' | 'reset' | 'button' | undefined
+  buttonType?: 'submit' | 'reset' | 'button'
```

Comment:
This isn't a biggy, but since the buttonType prop is used in multiple files, it might be good to turn it into a Enum.